src/algebra/big_operators/pi.lean

```diff
@@ -73,22 +72,26 @@ begin
   apply w,
 end
 
+@[ext]
```

Comment:
This should have
```suggestion
/-- This is used as the ext lemma instead of `add_monoid_hom.functions_ext` for reasons explained in
note [partially-applied ext lemmas]. -/
@[ext]
```
or some other wording linking to that note.